frontend/src/components/HelpDrawer/HelpDrawer.vue

```diff
@@ -9,6 +9,40 @@
       <!-- eslint-disable-next-line vue/no-v-html -->
       <div v-if="state.html" v-html="state.html"></div>
     </template>
+    <template #footer>
+      <div class="flex flex-row justify-center">
+        <aside v-if="locale === 'en-US'" class="w-1/2">
```

Comment:
Is there any special reason about using `aside` but not `div`? `aside` in semantic does not suitable for `footer`.